src/libsystemd/sd-netlink/generic-netlink.c

```diff
@@ -23,12 +28,12 @@ int sd_genl_socket_open(sd_netlink **ret) {
 static int lookup_id(sd_netlink *nl, sd_genl_family family, uint16_t *id);
 
 static int genl_message_new(sd_netlink *nl, sd_genl_family family, uint16_t nlmsg_type, uint8_t cmd, sd_netlink_message **ret) {
-        int r;
+        _cleanup_(sd_netlink_message_unrefp) sd_netlink_message *m = NULL;
         struct genlmsghdr *genl;
         const NLType *genl_cmd_type, *nl_type;
         const NLTypeSystem *type_system;
         size_t size;
-        _cleanup_(sd_netlink_message_unrefp) sd_netlink_message *m = NULL;
```

Comment:
Not strict we can put this in to reverse tree order to look good.

Reply:
Fixed.